fvtest/porttest/testHelpers.cpp

```diff
@@ -240,18 +240,16 @@ outputErrorMessage(struct OMRPortLibrary *portLibrary, const char *fileName, int
 	va_start(args, format);
```

Comment:
If the va_start/end moved into the callee then shouldn't we remove all usage inside the caller?